ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -807,6 +809,17 @@ else
 	mavlink start -r 800000 -d /dev/ttyACM0 -m config -x
 fi
 
+# XXX Do not run any mavlink instances since we need the serial port for communication with snapdragon
```

Comment:
This should be done according to this guide instead:
http://dev.px4.io/airframes-adding-a-new-frame.html